tests/ui_tests/cli/test_parameter_example.py

```diff
@@ -20,10 +20,16 @@
 from ert.mode_definitions import ENSEMBLE_EXPERIMENT_MODE
 from ert.plugins import ErtPluginManager
 from tests.unit_tests.config.egrid_generator import egrids
-from tests.unit_tests.config.summary_generator import names, summaries
+from tests.unit_tests.config.summary_generator import summaries
 
 from .run_cli import run_cli_with_pm
 
+names = st.text(
+    min_size=1,
+    max_size=8,
+    alphabet=st.characters(min_codepoint=65, max_codepoint=90),
```

Comment:
should we use `ord("A")` and `ord("Z")` for readability?

Reply:
Fixed